CHANGELOG.md

```diff
@@ -0,0 +1,330 @@
+# _fastlane_ CHANGELOG
+
+---
+
+When contributing to fastlane, don't forget to add an entry to this CHANGELOG.md file by copy/pasting and adapting this entry template:
+
+```
+* Describe your change here. End it with a full-stop and 2 spaces.  
```

Comment:
Alright, that makes sense! Never knew that this worked. This is a bit of an obscure requirement though, maybe it should be checked with Danger to make sure people follow it.